Casks/g/genymotion.rb

```diff
@@ -9,9 +9,11 @@
 
   livecheck do
     url "https://www.genymotion.com/download/"
-    regex(/Genymotion\s*Desktop\s*(\d+(?:\.\d+)+)/i)
+    regex(/genymotion-(\d+(?:\.\d+)+)\.dmg/i)
```

Comment:
```suggestion
    regex(/href=.*?Genymotion[._-]v?(\d+(?:\.\d+)+)\.(?:dmg|pkg)/i)
```

This suggestion is a variation on our typical pattern for matching the version from a filename in an `href` attribute. When it comes to casks, I sometimes include alternative filename extensions if it's using something like `dmg`, as the regex may fail to match a new version in the future if upstream switches to a different file format (e.g., pkg, zip, etc.). Upstream is using only the file extension to distinguish between files for macOS and Windows, so I've opted to just use the macOS-specific file extensions (dmg, pkg) for now.

Reply:
Thank you for the explanation, I'll try to remember it.

Reply:
Thank you for the explanation, I'll try to remember it.